### DISCUSSION TITLE
Cannot sync QFieldCloud project to Android: permission_denied_plan_insufficient

### DISCUSSION THREAD

**Discussion by max-mapper** [opengisch/QField#4385] [Bugs]
I am only using 5MB of my free 100MB. I get the error message: `permission denied becasue the useraccount's plan is insufficient`

![image](https://github.com/opengisch/QField/assets/39759/5ab71a8c-1f23-449b-92bf-3f410a590629)

**Comment by nealmaker** [comment]
I'm getting the same error, and only using 3.2 of my 100MB.

**Comment by Oisin-JK** [comment]
Me too - see #4409

**Comment by christabone** [comment]
I had the same error with my app -- it started after clearing the data & cache of the app in an attempt to force a refresh of an update from QFieldCloud.

 Unfortunately, the only way I was able to resolve this error was by making a copy of the project, deleting it from QFieldCloud, and remaking it again.

**Comment by L10ns** [comment]
Anybody knows if the problem was solved?
I have the same problem now.
The problem was occured about one month ago. Nothing change since before: no postgis files, no basemap layer added. I just added one light geopackage file. One day problem came out and started to report "network error...". Today I understood was for "permission_denied_plan_insufficient".
I collected some data now and I need to export to cloud and then to the pc.
I have a Community account and I'm not exceeding the 100 mb.
As I told already, nothing change before the error came.
Here my versions:
Qfield app: v. 3.4.4
Qfield Sync: v. 4.11.0

**Reply by nirvn** [reply]
@L10ns , can you type your cloud user name and project name here? If you feel uneasy doing that here, send over to mathieu@opengis.ch -- do _not_ send a password. Thanks.

**Comment by BoswachterMarc** [comment]
No problem with our community/volunteer project of 75MB...
(QGis 3.40 on Ubuntu 22.04.05 LTS, plugin QFieldSync 4.11.0 and QField 3.4.3 Ebo)

![afbeelding](https://github.com/user-attachments/assets/f3bd0628-0a27-4911-b5b2-14b1e57f456a)

**Comment by nordewal** [comment]
I have the same issue. It is a 33 MB project and does not use PostGIS, only local files. I tried removing the project and re-installing the app. Neither helped.

@nirvn the user is `galegge` and the project name `Galegge`. Any help would be appreciated!

**Reply by nirvn** [reply]
@nordewal , I'll have a look within the next 48 hours.

**Reply by SeqLaz** [reply]
Hi @nordewal ,

After investigating the job logs of your project, I found that the package synchronization failed because the project contains virtual layers, which are not allowed in the Community plan.

As recommendation is to exporting your virtual layers as a GeoPackage or duplicating the layers used to create the virtual layer. Then, directly filter your query in the Layer Properties > Source and use the "Provider Feature Filter" for querying.
![Screenshot from 2025-01-11 10-26-12](https://github.com/user-attachments/assets/ad9ef09c-0138-406d-91d7-467b7538127c)

**Reply by nordewal** [reply]
Thanks for investigating and for the recommendation on how to work around this issue. Given that most of our edits are in that base layer, it would be quite some work to always update/export that layer instead of using virtual ones.

I wasn't aware that virtual layers are not supported for community projects. Should this be added to the detailed comparison on the [pricing page](https://qfield.cloud/pricing.html)? Or even better, allow community projects to use virtual layers? :smile:

**Reply by SeqLaz** [reply]
@nordewal This feature is not currently supported by the community plan. However, it is in our backlog, and we plan to make it available for community plans in the future.

**Reply by nordewal** [reply]
That's awesome to know! Do you have something (FR, mailing list,...) I can follow along for when it's available?

**Comment by romanmeyerblch** [comment]
The pricing section could definitely use an update or a section with more details, where we can see what exactly is included in the different plans. I'm currently struggling with a project where I get the same error. Probably it's an issue with (too many?) wfs or url-datasets, but all I get is the 'insufficient plan' error. Quite frustrating.
